workflows/aspace_harvest.py

```diff
@@ -0,0 +1,81 @@
+import base64
+from datetime import datetime
+import json
+import os
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.mit import ECSOperator
+from airflow.sensors.mit import ECSTaskSensor
+
+
+network_config = json.loads(base64.b64decode(os.getenv('ECS_NETWORK_CONFIG')))
+cluster = os.getenv('ECS_CLUSTER')
+es_url = os.getenv('ES_URL')
+
+today = datetime.utcnow().strftime('%Y-%m-%d-%H-%M-%S')
+s3_file = "s3://aspace-oai-s3-stage/{file}.xml".format(file=today)
+
+
+def set_s3(**context):
+    context['ti'].xcom_push(key='aspace_oai_s3_key', value=s3_file)
```

Comment:
There's no need to mess with the context here. Returning a value from an operator automatically pushes it into XCOM. So you can rewrite this function to something like:
```python
def set_s3():
    today = datetime.utcnow().strftime('%Y-%m-%d-%H-%M-%S')
    return "s3://aspace-oai-s3-stage/{file}.xml".format(file=today)
```